pkg/perfcounter/counter.go

```diff
@@ -15,43 +15,42 @@
 package perfcounter
 
 import (
-	"sync/atomic"
-
 	"github.com/matrixorigin/matrixone/pkg/common/mpool"
+	"github.com/matrixorigin/matrixone/pkg/util/metric/stats"
 )
 
 type Counter struct {
 	S3 struct {
-		List        atomic.Int64
-		Head        atomic.Int64
-		Put         atomic.Int64
-		Get         atomic.Int64
-		Delete      atomic.Int64
-		DeleteMulti atomic.Int64
+		List        stats.Counter
+		Head        stats.Counter
+		Put         stats.Counter
+		Get         stats.Counter
+		Delete      stats.Counter
+		DeleteMulti stats.Counter
 	}
 
 	Cache struct {
-		Read     atomic.Int64
-		Hit      atomic.Int64
-		MemRead  atomic.Int64
-		MemHit   atomic.Int64
-		DiskRead atomic.Int64
-		DiskHit  atomic.Int64
+		Read     stats.Counter
+		Hit      stats.Counter
+		MemRead  stats.Counter
+		MemHit   stats.Counter
+		DiskRead stats.Counter
+		DiskHit  stats.Counter
 	}
 
 	DistTAE struct {
 		MPool mpool.MPoolStats
```

Comment:
I am still not quite sure if we should put mpoolstats here.    Not clean to me how do we use this ...

Reply:
@reusee Please share your thoughts.